.gitlab-ci.yml

```diff
@@ -219,24 +219,17 @@ e2e:
   interruptible: true
   artifacts:
     when: always
-    paths: ['test-report/e2e/specs.log']
+    paths:
+      - 'test-report/e2e/*'
     reports:
       junit: test-report/e2e/*.xml
   script:
     - yarn
+    - yarn playwright install --with-deps
```

Comment:
Unfortunately playwright recommends to [NOT cache browsers](https://playwright.dev/docs/ci#caching-browsers)

This is not ideal because it takes about as much time to install dependencies and browsers than running the tests in the CI.
I believe we can get around that by having pre-build Docker image.